README.md

```diff
@@ -13,51 +13,26 @@ familiar with these before editing.
 To run the site locally with Docker, use the following command:
 
 ```bash
-docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll  -it -p 127.0.0.1:4000:4000 jekyll/jekyll sh -c "bundle install && jekyll serve"
+docker run --label=jekyll --volume=$(pwd):/srv/jekyll  -it -p 127.0.0.1:4000:4000 jekyll/jekyll:3.5.2 sh -c "bundle install && rake test && jekyll serve"
 ```
 
-Make sure you are not introducing html errors or bad links:
-```bash
-docker run --rm --label=jekyll --volume=$(pwd):/srv/jekyll  -it  jekyll/jekyll sh -c "bundle install && rake test"
-```
+The `rake test` part is to make sure you are not introducing html errors or bad links, you should see
 ```
 HTML-Proofer finished successfully.
 ```
+in the output
 
-#### Side note for those on non-linux machines
```

Comment:
I don't think we need to tell people how to install docker and the minikube part is just noise for this docs repo